lib/DependencyProvider.js

```diff
@@ -162,13 +177,14 @@ class DependencyProvider {
     extra /*: { [string]: string } */
   ) /*: string */ {
     const lister = new OfflineAvailableVersionLister();
+    const indirectDeps = JSON.parse(elmJson).dependencies?.indirect;
     try {
       return wasm.solve_deps(
         elmJson,
         useTest,
         extra,
         fetchElmJsonOffline,
-        (pkg) => lister.list(pkg)
+        (pkg) => lister.list(pkg, indirectDeps)
```

Comment:
Assuming that `elmJson` is a well-formed elm.json, the type of `.dependencies.indirect` must be `{ [string]: string }`, right? (A plain JS object.) But `list.list` says that it wants `Map<string, string>`. (A `Map` instance.) And the code uses `.get()`. So maybe when you tested this in the example folder, you weren’t testing what you thought? Or maybe I’m missing something.